server_api.py

```diff
@@ -1176,7 +1176,7 @@ def change_password(json_obj):
                 # old password correct
                 hashed_key = bcrypt.hashpw(new_password.encode('utf-8'),bcrypt.gensalt())
                 with UserDao() as userDao:
-                    userDao.updatePassword(hashed_key=str(hashed_key)[1:],userId=str(user_id))
+                    userDao.updatePassword(hashedKey=str(hashed_key)[2:-1],userId=str(user_id))
```

Comment:
BTW, could you help to change the error message in line 1183~1184
help modifying to
```
except Exception as e:
    return_msg["error"] = str(e)
```
Because the error is nothing to do with `no such user id`, it has checked in line 116~1167